Cargo.toml

```diff
@@ -1,6 +1,7 @@
 [workspace]
 resolver = "2"
 members = ["xtask/", "lib/*", "crates/*"]
+exclude = ["crates/proc_macro_test/imp"]
```

Comment:
Do we really need this? I think the fact that `imp/Cargo.toml` has `[workspace]` should be enough.

Reply:
That's what I tried before and it resulted in a Cargo error

`error: multiple workspace roots found in the same workspace`